src/DacpacTool/PackageBuilder.cs

```diff
@@ -42,7 +42,18 @@ public void AddExternalReference(string referenceFile, string externalParts)
             ValidateReference(referenceFile);
 
             Console.WriteLine($"Adding reference to {referenceFile} with external parts {externalParts}");
-            Model.AddReference(referenceFile, externalParts);
+            Model.AddReference(referenceFile, externalParts, false);
+        }
+
+        public void AddExternalReference(string referenceFile, string externalParts, bool suppressErrorsForMissingDependencies)
```

Comment:
We now basically have three overloads that all have almost exactly the same implementation. Perhaps we should just add defaults to the overload with the most parameters and remove the other overloads now.

Reply:
Yea no prob, i am more of a fan of defaulted parameters anyway.  I just was following the overload pattern because it was established.  But i guess you now have the rule of 3.  Will push back up later today.